### PR TITLE
updated specs

### PR DESCRIPTION
**Pull request by defektive**
this also sets up skydrive to have its own raven dsn

### CODE REVIEW

File: lib/skydrive/client.rb [hunk 1]
```diff
@@ -180,6 +183,7 @@ def api_call(url, headers = {})
         }
         c.perform
 
+        buffer = "<notjson />"
```
Comment (rivernate) [inline lib/skydrive/client.rb]: what is this for?

Comment (rivernate) [inline lib/skydrive/client.rb]: why are we reassigning buffer?

File: lib/skydrive/file.rb [hunk 2]
```diff
@@ -33,7 +33,7 @@ def update_content_type_data(allowed_extensions=nil)
         self.kind = mm.comment
         self.suffix = mm.extensions.last
 
-        if allowed_extensions.empty? || (allowed_extensions & mm.extensions).size > 0
+        if (allowed_extensions == nil || allowed_extensions.empty?) || (allowed_extensions & mm.extensions).size > 0
```
Comment (rivernate) [inline lib/skydrive/file.rb]: you can use `allowed_extensions.blank?` here instead

File: lib/skydrive/client.rb (hunk 1 above)
Comment (defektive) [inline lib/skydrive/client.rb]: my bad, that was my way making sure my exception handling worked

File: lib/skydrive/file.rb (hunk 2 above)
Comment (defektive) [inline lib/skydrive/file.rb]: nice. I thought there was something that handled both empty and nil.

**Pull request merged**